src/components/Registration/BusinessTypeConfirm.vue

```diff
@@ -1,32 +1,36 @@
 <template>
   <div id="business-type-confirm">
-    <v-row no-gutters>
-      <v-col cols="12" sm="3" class="pr-4">
-        <label class="d-block">Business Type</label>
-        <v-chip
-          v-if="hasBusinessTypeChecked"
-          id="checked-chip"
-          x-small label
-          color="primary"
-          text-color="white"
-        >
-          Checked
-        </v-chip>
-      </v-col>
-
-      <v-col cols="12" sm="9">
-        <p class="mb-0 checkbox-gp-text">{{ label }}</p>
-        <div id="business-check-div">
-          <v-checkbox
-            class="mt-0"
-            v-model="checked"
-            hide-details
-            :label="text"
+    <div class="section-container">
```

Comment:
Added div tag with "section-container" class to wrap the v-row tag to accommodate for the component with the correct padding.

Reply:
I'm sure this works but it's bad practise to have whitespace around the component in the component itself: that whitespace might not work if you used this component on another page. It's better to specify the whitespace in the parent. (The parent's layout is the parent's problem to solve, not this component's.)

Reply:
This div tag is removed in the last commit: fa7c46a.